src/cmds/restic/cmd_snapshots.go

```diff
@@ -87,6 +86,22 @@ func runSnapshots(opts SnapshotOptions, gopts GlobalOptions, args []string) erro
 
 	}
 
+	if opts.Json == true {
+		jsonoutput(list)
+	} else {
+		naturaloutput(list)
+	}
+
+	return nil
+}
+
+//naturaloutput provides human redability
+func naturaloutput(list []*restic.Snapshot) {
```

Comment:
This function needs a new name, all functions in the restic binary share the same namespace. What about `printSnapshots()`?